### DISCUSSION TITLE
Improve project intuitiveness

### DISCUSSION THREAD

**Discussion by jayhamilton** [community/community#10443] [Projects and Issues]
Connecting issues to projects and projects to repositories is a bit unintuitive. I think my comment may just apply to the Beta version of projects? I am new to this and don't have experience with the prior version of how Projects worked. Anyway, my first attempt at using projects was confusing. I finally sat down today and tried to figure out how to link things up. The primary issue initially was that I expected to be able to setup the project more directly from within the repository view. Jumping out of that view to go to a separate projects view was what threw me off. Now that I have the projects connected to the repository, what now seems to be confusing is that if I go to the repository issues, the display indicates I don't have a project and I get no sense that the issues are connected to the project they are defined within. Also, it would be helpful if, once in the repository and there is an associated project, why do I have to select it to display when I click the repository tab. I have only one project and maybe that one should be displayed by default when I select the project tab at the top.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
